style.css

```diff
@@ -0,0 +1,51 @@
+header {
```

Comment:
appreciate the consistent design system across your entire site with the global style sheet!